### PR TITLE
✨ Building home page

### PR DESCRIPTION
**Pull request by fabiglahn**
## Description
I used the Card Component to build my home page, that afterwards is supposed to link the four different categories, that are yet to be built. 

## Design
Bildschirmfoto 2021-12-01 um 15 54 05

### CODE REVIEW

**Review by AnkePfei2fer — APPROVED** [review]
lgtm, good work! 💯
Just a personal design opinion in addition: If you would place the home cards more at the bottom, peoples faces in the background image would be visible, which I find more appealing than hiding them

File: src/Components/HomeCards/HomeCards.module.css [hunk 1]
```diff
@@ -2,7 +2,8 @@
   display: grid;
   grid-template-columns: 50% 50%;
   grid-template-rows: 50% 50%;
-  width: 100%;
+  width: 100vw;
+  height: 100vw;
```
Comment (marwinburesch) [inline src/Components/HomeCards/HomeCards.module.css:6]: Too many Volkswagens mate, maybe there is a similar unit that goes for height 🤔

File: src/Pages/Home/Home.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import react from "react";
```
Comment (marwinburesch) [inline src/Pages/Home/Home.tsx]: That 'r' seems to small ;)

File: src/Pages/Home/Home.module.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+  height: 100vh;
+}
+
+img {
+  opacity: 0.5;
+}
```
Comment (marwinburesch) [inline src/Pages/Home/Home.module.css]: What is 'src/Assets/background_opacity.png' for, then?

**Review by marwinburesch — COMMENTED** [review]
🚗 
Awesome though!

File: src/Components/HomeCards/HomeCards.module.css (hunk 1 above)
Comment (fabiglahn) [reply src/Components/HomeCards/HomeCards.module.css:6]: This was my way to try to keep the squares square, no matter the size of the screen. I have the same issue again with another component now. Tried like a million different solutions. Do you have an idea?

**Pull request merged**